modules/nf-core/foldcomp/decompress/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+nextflow_process {
+
+    name "Test Process FOLDCOMP_DECOMPRESS"
+    script "../main.nf"
+    process "FOLDCOMP_DECOMPRESS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "foldcomp"
+    tag "foldcomp/compress"
+    tag "foldcomp/decompress"
+
+    setup {
+        run("FOLDCOMP_COMPRESS") {
+            script "../../compress/main.nf"
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['proteomics']['pdb']['tim1_pdb'], checkIfExists: true)
+                    ]
+                """
+            }
+        }
+    }
+
+    test("proteomics - pdb") {
+
+        when {
+            process {
+                """
+                input[0] = FOLDCOMP_COMPRESS.out.fcz.flatMap { meta, files ->
+                    files.collect { file -> tuple(meta, file) }
+                }
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out[0][0][1]).match('testA_0') },
+                { assert snapshot(process.out[0][1][1]).match('testA_1') },
+                { assert snapshot(process.out[0][2][1]).match('testB_0') },
+                { assert snapshot(process.out[0][3][1]).match('testB_1') },
+                { assert snapshot(process.out.versions).match('versions') }
```

Comment:
I'm assuming you are specifying individual outputs since there are inconsistent hashes for certain files. If this is the case, perhaps you could include some assertions for file content or to check if the file exists?

Reply:
I sorted the output files and now a whole snapshot works.